openvpn.h

```diff
@@ -30,11 +30,25 @@
     {                                                                                            \
         if (SetPropW(hwnd, name, p))                                                             \
             break;                                                                               \
-        MsgToEventLog(EVENTLOG_ERROR_TYPE, L"%hs:%d GetProp returned null", __func__, __LINE__); \
+        MsgToEventLog(EVENTLOG_ERROR_TYPE, L"%hs:%d SetProp returned null", __func__, __LINE__); \
         EndDialog(hwnd, IDABORT);                                                                \
         return false;                                                                            \
     } while (0)
 
+/* A macro to set lval = GetProp(hwnd, name) or log an error and return err_return */
+#define TRY_GETPROP(hwnd, name, lval, err_return)                                          \
+    do                                                                                     \
+    {                                                                                      \
+        HANDLE handle_##lval = GetProp(hwnd, name);                                        \
```

Comment:
Inside a block, any reasonably unique name would do for a local variable. Concatenation here is an unnecessary complication.